_templates/exchanges.html

```diff
@@ -251,6 +251,8 @@ <h3 id="mexico" class="no_toc">Mexico</h3>
           <a class="marketplace-link" href="https://bitso.com/">Bitso</a>
           <br>
           <a class="marketplace-link" href="https://www.volabit.com/">Volabit</a>
+          <br>
```

Comment:
Exchanges should be added in alphanumeric order.

Reply:
Hello, thanks for your comment. We committed a change where the alphanumeric order is applied.